configs/config.protectli_vp4620

```diff
@@ -0,0 +1,29 @@
+CONFIG_LOCALVERSION="1.0.13"
+CONFIG_VENDOR_PROTECTLI=y
+CONFIG_CBFS_SIZE=0xa00000
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x20000
+CONFIG_USE_LEGACY_8254_TIMER=y
+CONFIG_HAVE_IFD_BIN=y
+CONFIG_BOARD_PROTECTLI_VP4620=y
+CONFIG_TIANOCORE_BOOTSPLASH_FILE="$(top)/3rdparty/blobs/mainboard/$(MAINBOARDDIR)/bootsplash.bmp"
+CONFIG_UART_PCI_ADDR=0x0
+CONFIG_HAVE_ME_BIN=y
+CONFIG_ME_REGION_ALLOW_CPU_READ_ACCESS=y
+CONFIG_SUBSYSTEM_VENDOR_ID=0x0000
+CONFIG_SUBSYSTEM_DEVICE_ID=0x0000
+CONFIG_SMMSTORE_V2=y
+CONFIG_SMMSTORE_SIZE=0x40000
+CONFIG_BMP_LOGO=y
+CONFIG_FSP2_0_LOGO_FILE_NAME="$(top)/3rdparty/blobs/mainboard/$(MAINBOARDDIR)/bootsplash.bmp"
+CONFIG_TPM2=y
+CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_DEFAULT_CONSOLE_LOGLEVEL_0=y
+CONFIG_POST_DEVICE_LPC=y
+CONFIG_POST_IO_PORT=0x80
+CONFIG_PAYLOAD_TIANOCORE=y
+CONFIG_TIANOCORE_CUSTOM=y
+CONFIG_TIANOCORE_REPOSITORY="https://github.com/Dasharo/edk2.git"
+CONFIG_TIANOCORE_TAG_OR_REV="7f90b9cd8df803d1e4609d12a30a1bf1658ab018"
+# CONFIG_TIANOCORE_HAVE_EFI_SHELL is not set
+CONFIG_TIANOCORE_CUSTOM_BUILD_PARAMS="-D PLATFORM_BOOT_TIMEOUT=6 -D BOOT_MENU_KEY=0x0015 -D SETUP_MENU_KEY=0x0008 -D BOOTLOADER=COREBOOT -D PCIE_BASE=0xc0000000 -DPS2_KEYBOARD_ENABLE -D NETWORK_IPXE=TRUE -D SERIAL_TERMINAL=TRUE -D TPM_ENABLE=TRUE -D NETWORK_ENABLE=TRUE"
```

Comment:
Are we going to use this Kconfig option instead of custom options from now on? I don't mind either way, and it's what i did for the thinkcentre too, just asking.

Reply:
It will be easier to align with upstream since custom tianocore build option is allowed. Thus I thought about such solution.